src/components/AddToWatchListButton/AddToWatchListButton.jsx

```diff
@@ -0,0 +1,93 @@
+
+import { useState } from 'react'
+import { useDispatch } from 'react-redux';
+import style from './AddToWatchListButton.module.css'
+import { toWatchList } from '../../store/filmApi/types';
+
+const buttonTypeArr = {
```

Comment:
Helloo ) Why `Arr` if it's object. And it's necessary to add type of var to its name :)